src/main/scripts/org/reaktivity/specification/http/rfc7230/connection.management/request.with.header.exclude/server.rpt

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2016-2020 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property serverTransport "nukleus://streams/target#0"
+
+accept ${serverTransport}
+  option nukleus:window 8192
+  option nukleus:transmission "duplex"
+accepted
+connected
+
+read "OPTIONS / HTTP/1.1" "\r\n"
+read "Host: localhost:8080" "\r\n"
+read "Access-Control-Request-Headers: origin" "\r\n"
+
+read "\r\n"
+
+write "HTTP/1.1 200 OK\r\n"
+write "Content-Length: 0\r\n"
+write "Access-Control-Allow-Headers: origin\r\n"
+write "\r\n"
```

Comment:
The CORS preflight test and the regular request should be separate script scenarios because they are two different usecases, even though they might typically occur back to back in practice.